### DISCUSSION TITLE
OIDC use client_secret_post (post form) or client_secret_basic (header)

### DISCUSSION THREAD

**Discussion by felixilgatto** [Budibase/budibase#12371] [Help] [answered]
Hello everyone,
I would like to know if OIDC use client_secret_post (post form) or client_secret_basic (header) for client authentication.

Thank you ❤️

**Comment by ConorWebb96** [comment, marked as answer]
Hey @felixcapuano,

Yes it uses the header for client authentication. Here is the library that we use. https://www.npmjs.com/package/@techpass/passport-openidconnect

i hope this helps.

**Comment by felixilgatto** [comment]
Thank you for your response !
